src/main/java/com/actionlistapi/model/ActionRequest.java

```diff
@@ -19,140 +19,136 @@
 @Entity
 public class ActionRequest {
 	
-	@Id
-	@GeneratedValue(strategy = GenerationType.IDENTITY,generator="id_seq")
-	@SequenceGenerator(name="id_seq",sequenceName="id_seq",allocationSize=1)
-	private long id;
+	@Id 
```

Comment:
It looks like there's still a space after `@Id` in the most recent version of the code.